src/Physics/babylon.physicsEngine.ts

```diff
@@ -195,7 +195,7 @@
         sleepBody(impostor: PhysicsImpostor);
         wakeUpBody(impostor: PhysicsImpostor);
         //Joint Update
-        updateDistanceJoint(joint: DistanceJoint, maxDistance:number, minDistance?: number);
+        updateDistanceJoint(joint: PhysicsJoint, maxDistance:number, minDistance?: number);

```

Comment:
Thanks @RaananW for the merge.

Just to answer you. I did this because it is the best common class to be used by both Cannon and Oimo plugins. Cannon plugin wants to use DistanceJoint in its updateDistanceJoint method parameter while Oimo wants to use IMotorEnabledJoint. PhysicsJoint class is the middle ground as it is the base class of DistanceJoint and the only implementer class of IMotorEnabledJoint interface.

I hope this clarifies things up :)